tests/baselines/reference/jsDeclarationsGetterSetter.js

```diff
@@ -154,3 +267,18 @@ export class G {
 export class H {
     set x(value: any);
 }
+export class I {
+    set x(value: number);
+}
+export class J {
+    set x(value: number);
+}
+export class K {
+    set x(value: number);
+}
+export class L {
+    set x(value: any);
+}
+export class M {
+    set x(value: any);
+}

```

Comment:
This one could create a "unified" signature of
```suggestion
export class M {
    set x(value: boolean);
}
```
but that's tangent and not a regression.